test/clojure/core_test/mapcat.cljc

```diff
@@ -56,4 +56,7 @@
                    (mapcat 42 [1 2]))))
     (testing "non-concatable return value"
       (is (thrown? #?(:cljs :default :default Exception)
-                   (mapcat identity (range 2)))))))
+                   ;; we need to realize the lazy data structure
+                   ;; in order for `cljs` to throw on the result
+                   ;; hence `(into [])`
+                   (into [] (mapcat identity (range 2))))))))
```

Comment:
This is what `doall` is for.

Reply:
Do you prefer `doall` to realizing a vector, they both work here (I don't have a preference)?

Reply:
Yes, please use `doall`. We don't want a vector, we just want to force the lazy seq to be realized. That is the exact purpose of `doall`. That also allows us to remove the comment, since `doall` being single purpose makes its usage self-documenting.